lms/templates/certificates/_accomplishment-header.html

```diff
@@ -6,7 +6,7 @@
 
     <header class="header-app" role="banner">
         <h1 class="header-app-title">
-            <a class="logo" href="${request.site}" style="width: auto;">
+            <a class="logo" href="https://${request.site}" style="width: auto;">
```

Comment:
True, `request.site` would break.

Why not just `/`?

```suggestion
            <a class="logo" href="/" style="width: auto;">
```

Reply:
hahaha, no reason @OmarIthawi . Just the too-fast-too-dumb mode 😆